docs/adr/0014-auth-for-settings-redesign.md

```diff
@@ -0,0 +1,95 @@
+# Authentication in Settings Redesign
+
+- Status: proposed
+- Deciders: Les Orchard, Lauren Zugai, Ben Bangert
+- Date: 2020-04-15
+
+Issue: https://jira.mozilla.com/browse/FXA-1559
+
+## Context and Problem Statement
+
+The Settings Redesign app needs to query & mutate the same protected user data as the existing settings app hosted on content-server. This will require some form of authentication & authorization to manage that data.
+
+## Decision Drivers
+
+- Smooth UX
+- Security
+- Development velocity
+- Ease of integration
+
+## Considered Options
+
+- Reuse existing sessionToken on content-server with auth-server APIs
+- Authenticate via OAuth2 to use auth-server APIs
+- Authenticate via OAuth2 to use DB directly
+- Implement own login sessions
+
+## Decision Outcome
+
+Chosen option: "Reuse existing sessionToken on content-server with auth-server APIs", because it will have the least impact on existing auth-server implementation and infrastructure outside the Settings Redesign app in general.
+
+### Positive Consequences
+
+- The new Settings Redesign app can reuse existing auth-server APIs in the same way as existing settings code. This can constrain the novelty in the project to the redesigned & reimplemented settings UX.
+
+### Negative Consequences
+
+- We don't have an entirely greenfield project, which could be a bit of a drag. But, rediscovering lessons learned in existing code can also be a drag.
+
+## Pros and Cons of the Options
+
+### Reuse existing sessionToken on content-server with auth-server APIs
+
+The Settings Redesign frontend will be hosted at `accounts.firefox.com/beta/settings` per [FXA-1539](https://jira.mozilla.com/browse/FXA-1539) / [#4819](https://github.com/mozilla/fxa/issues/4819). This means it will have access to the same credentials as the original settings frontend on content-server.
+
+- Pros:
+  - Requires few or no auth-server changes - APIs already expect sessionToken
+  - Can be done with seamless UX - user visits new settings without confirmation
+- Neutral:
+  - GraphQL server needs to proxy auth-server APIs
+- Cons:
+  - None? Need to be as careful with the session token as existings settings app.
+
+[code-via-cookie]: https://github.com/shane-tomlinson/fxa/commits/i881-payments-using-oauth-code-using-cookie
```

Comment:
I don't see any references to this footnote, is it left over from a previous iteration?

Reply:
Yeah, that shouldn't be there any more. Will delete.